Lesson21/duygu/todo-app/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>To-Do App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main class="container">
+      <form class="todo-form" id="todoForm">
+        <label for="todoInput">To do Item</label>
+        <input type="text" placeholder="Add to do item" id="todoInput" />
+        <button type="submit">SUBMIT</button>
```

Comment:
'Add' would be a more descriptive name for the action. Once the user typed a todo into input field, they can add it.
```suggestion
        <button type="submit">Add</button>
```